### PR TITLE
[RDY] Gerlof merge interesting

### PR DESCRIPTION
**Pull request by gfokkema**
Comes after #172

### CODE REVIEW

File: dnainator-core/src/main/java/nl/tudelft/dnainator/graph/interestingness/Scores.java [hunk 1]
```diff
@@ -4,7 +4,28 @@
...
+		public int applyImportanceModifier(int rawScore) {
+			if (rawScore < SEQ_LENGTH_THRESHOLD) {
+				return rawScore;
+			} else {
+				return rawScore * rawScore;
+			}
+		}
+	},
+	DR_MUT("Drug Resistance Mutation Score") {
+		private final int[] multipliers = { 0, 800, 1000 };
```
Comment (Balletie) [inline dnainator-core/src/main/java/nl/tudelft/dnainator/graph/interestingness/Scores.java]: Should maybe be static?

**Comment by Unia** [comment]
LGTM :+1:

File: dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/ClusterDrawable.java [hunk 2]
```diff
@@ -103,20 +104,24 @@ private void initProperties() {
...
 		properties.put(ClusterPropertyTypes.SEQUENCE, sn.getSequence());
+		properties.put(ClusterPropertyTypes.SCORE, score);
+
+		sn.getScores().forEach((k, v) -> properties.put(k, Integer.toString(v)));
 	}
 
 	private void draw(ColorServer colorServer) {
 		double radius = getRadius();
 
+		Circle commonNode = new Circle(radius);
```
Comment (pvanagtmaal) [inline dnainator-javafx/src/main/java/nl/tudelft/dnainator/javafx/drawables/strains/ClusterDrawable.java]: No need for this. Should be put back.

**Comment by pvanagtmaal** [comment]
Nice, merging :)

**Pull request merged**